apps/fabric-website/src/pages/Controls/ButtonPage/docs/android/ButtonOverview.md

```diff
@@ -0,0 +1,5 @@
+Buttons are one of the core experiences that help make our products feel native to the platform they’re on. It’s important that we resepect this paradigm in an effort to keep our design quality high and at home on Android.
```

Comment:
This wording seems oriented towards internal users (and is generally a bit awkward).

Reply:
I hear you there, I adapted it slightly from the toolkit description. Did you pull the other descriptions from somewhere or make it up? I assumed we'd want some kind of thought out HIG oriented description at some point.

Reply:
@jordandrako might have a better idea about where the descriptions came from--I mostly just copied things that were already in the HIG.

Reply:
Agreed this could be tweaked a bit. The original descriptions came over the last few years as a collaboration between designers who borrowed much of the guidance from existing guides like the Windows developer docs.

Back to this PR...

First, some minor typos:
- natvie -> native
- resepect -> respect

Second, to @ecraig12345 's observation, I think there's a larger editorial point that we maybe don't need to resolve here in this PR: Tonally, this does feel a little odd and inconsistent with other content on the site, especially compared to the Web's content. For example, here we're using a lot of "our" or "we"-style language to describe–almost defend–the design of the control, as opposed to guidance about what the button does or how it should be used. I understand this is what the toolkits say, and perhaps we should leave those as-is, but this feels incongruent with the website right now.

I wonder if we strike a balance in the short term by doing something like this:
> Buttons are one of the core controls that make an app feel native to the platform it's on. It’s important to respect the platform's paradigms in order to help the user at home on iOS and keep the experience quality high.

Thoughts?

cc @bigbadcapers who may already be thinking about this for a future update :)

Reply:
Definitely agree that we need to do an all-up tone/style pass on the website content. I like @Jahnp's suggested alternative text for now.

Reply:
Thanks for the feedback and direction, @Jahnp and @ecraig12345 ! The context definitely helps. I'll double check with @bigbadcapers to see if he has anything to add.

Reply:
Just chatted offline with @bigbadcapers - I think we'll go with your proposal for now, Peter, but yes, there is a general plan to do a pass on content at some point.